curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/nest-an-anchor-element-within-a-paragraph.english.md

```diff
@@ -76,7 +76,17 @@ tests:
 ## Solution
 <section id='solution'>
 
-```js
-// solution required
+```html
+<h2>CatPhotoApp</h2>
+<main>
+  
+  <a href="http://freecatphotoapp.com" target="_blank">cat photos</a>
```

Comment:
Based on the challenge instructions, this original anchor element should not be here anymore.  The instructions said to nest the existing `a` element (which is this one).
```suggestion
```

We may need to think about another PR which changes the wording of the instructions for this challenge to something like:
>Now move the existing `a` element so that it is nested within a new p element (just after the existing main element). 

Then the tests would need to validate the first child of the`main` element is a the new `p` element with the anchor element in it.  Also, the tests would need to validate the anchor element is not a child of the `main` element anymore.

Reply:
@RandellDawson All right. How about I amend this PR to remove the sole `a` element on Line 83, so that it's easier to get this one merged rightaway? Then I can create another one to change the challenge wording and the tests.

Also, about this - 

> Then the tests would need to validate the first child of themain element is a the new p element with the anchor element in it.

This would mean that the `p` and the nested `a` elements should be part of the `main` element. 

But, based on the current instructions: 

> Now nest your existing a element within a new p element (just after the existing main element)

I thought it asked the user to move the `p` and `a` elements to outside the `main` element, which is where I placed the code (Line 90). Even if I'm wrong, the instructions are ambiguous and it would be good to explicitly specify if the `p` and `a` should be placed **inside** the `main` or **outside** it. 

If we decide to move it inside the `main` (I prefer this), I'll change line 90 to move it in `main` in this PR.

Reply:
I think the intent of the original instruction (just after the existing main element) probably meant to say (just after the opening main tag) which would mean inside the main element (what you prefer).  I am all for this change.